### DISCUSSION TITLE
Add album art in Fre:ac

### DISCUSSION THREAD

**Discussion by neshera** [enzo1982/freac#273] [General Discussion]
When ripping a CD, anyone know how to add an album art image to files within Fre:ac?
I am here as a "refugee" from Max, which I loved, but cannot use with current version of iOS. I am using Kid3 tag editor to add the image files, but would be nice to do it while converting within Fre:ac.
Thank you all in advance!

**Comment by enzo1982** [comment]
Go to the _Tags_ tab, there you can add album art. The tag editor has two modes, _Track_ and _Album_ mode. In album mode you can add the same image to all tracks belonging to a CD at once.

I'm also working on automatic album art downloads. But cannot yet say for sure when this will be available.
